style.css

```diff
@@ -1014,3 +1019,235 @@ button:active {
   text-decoration: none;
   padding-top: 20px;
 }
+
+/* media query */
+@media screen and (min-width: 768px) {
+  .main-mobile {
+    display: flex;
+    width: auto;
+  }
+
+  .nav-links ul li {
+    display: inline;
+    color: #36b37e;
+    flex-direction: row;
+    justify-content: space-between;
+    justify-items: center;
+    left: 130%;
+  }
+
+  #menu {
+    display: none;
+  }
+
+  #welcome {
+    display: none;
+  }
+
+  .headline {
+    background-image: url(/images/Header-llustration-desktop@2x.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 160px 0;
+    position: absolute;
+    width: 100%;
+  }
+
+  .black {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+    text-align: center;
+    letter-spacing: 0.37px;
+    color: #172b4d;
+    margin-top: -5%;
+  }
+
+  .text {
+    width: 720px;
+    height: 96px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #172b4d;
+    margin-top: -8%;
+  }
+
+  #text2 {
+    width: 947px;
+    margin-top: -5%;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+
+    /* or 130% */
+
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  .card-workrow {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    padding: 0;
+    margin-left: 5px;
+    width: 1171px;
+    height: 988px;
+  }
+
```

Comment:
- [ ] Please do the same in this section. Just as requested by the previous reviewer. Please try to refer to the explanation I have given in the other comments 😉